app/invocation/invocation.tsx

```diff
@@ -37,7 +36,7 @@ interface State {
 interface Props {
   user?: User;
   invocationId: string;
-  hash: string;
+  hash: InvocationTabHash;
```

Comment:
This `hash` value can be any string though right - since it just comes from the users hash. Does casting it as a type hide that fact that you have to handle arbitrary strings throughout the code?